<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History</class>
 <widget class="QDialog" name="History">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historia PCB</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog#History {
    background-color: #2f3136;
    color: #e8e9ea;
    font-family: "Segoe UI", Roboto, Cantarell, "Helvetica Neue", sans-serif;
    font-size: 14px;
}

/* --- Pole Wyszukiwania --- */
QLineEdit#search_field {
    background-color: #40444b;
    color: #dcddde;
    border: 1px solid #282a2e;
    border-radius: 5px;
    padding: 8px 12px;
    font-size: 16px;
    min-height: 36px;
}
QLineEdit#search_field:focus {
    border-color: #5865F2;
}

/* --- Widok Tabeli --- */
QTableView#tableWidget {
    background-color: #36393f;
    border: 1px solid #282a2e;
    border-radius: 5px;
    gridline-color: #4f545c;
    color: #dcddde;
    font-size: 15px;
    selection-background-color: #5865F2;
    selection-color: #ffffff;
    outline: none;
    alternate-background-color: #40444b;
}
QTableView#tableWidget::item {
    padding: 8px 10px;
    border-bottom: 1px solid #4f545c;
}

QTableView#tableWidget::item:selected {
    background-color: #5865F2;
    color: #ffffff;
    border-radius: 3px;
    font-weight: 600;
}

QTableView#tableWidget::item:hover:!selected {
    background-color: #4f545c;
    border-radius: 3px;
}

/* --- Nagłówki Tabeli --- */
QHeaderView::section {
    background-color: #2f3136;
    color: #ffffff;
    padding: 10px 8px;
    border: none;
    border-bottom: 1px solid #4f545c;
    font-size: 14px;
    font-weight: 600;
}
QHeaderView::section:horizontal {
    border-right: 1px solid #4f545c;
}
QHeaderView::section:horizontal:last {
    border-right: none;
}
QHeaderView::section:vertical {
    background-color: #40444b;
    border-right: 1px solid #4f545c;
    color: #b9bbbe;
    font-weight: 500;
    padding-right: 12px;
}
QTableView QTableCornerButton::section {
    background-color: #2f3136;
    border: none;
    border-bottom: 1px solid #4f545c;
    border-right: 1px solid #4f545c;
}

/* --- Dolny Pasek (Stopka) --- */
QFrame#bottomBar {
    background-color: #36393f;
    border: none;
    border-top: 1px solid #282a2e;
    min-height: 45px;
    max-height: 45px;
}

/* --- Przycisk Powrotu --- */
QPushButton#backButton {
    background-color: #5865F2;
    color: white;
    font-size: 13px;
    font-weight: 600;
    padding: 6px 20px;
    border-radius: 5px;
    border: none;
    min-height: 28px;
    outline: none;
}
QPushButton#backButton:hover { background-color: #4e5dcf; }
QPushButton#backButton:pressed { background-color: #404ab8; }
QPushButton#backButton:disabled { background-color: #40444b; color: #72767d; opacity: 0.7; }

/* --- Paski Przewijania --- */
QScrollBar:vertical { border: none; background: #2f3136; width: 12px; margin: 0px; }
QScrollBar::handle:vertical { background: #4f545c; min-height: 25px; border-radius: 6px; }
QScrollBar::handle:vertical:hover { background: #5a606b; }
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical { border: none; background: none; height: 0px; }
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical { background: none; }

QScrollBar:horizontal { border: none; background: #2f3136; height: 12px; margin: 0px; }
QScrollBar::handle:horizontal { background: #4f545c; min-width: 25px; border-radius: 6px; }
QScrollBar::handle:horizontal:hover { background: #5a606b; }
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal { border: none; background: none; width: 0px; }
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal { background: none; }

   </string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout"> <property name="spacing">
    <number>10</number> </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLineEdit" name="search_field">
     <property name="placeholderText">
      <string>Wyszukaj numer PCB...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableWidget">
     <property name="sizePolicy"> <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch> </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottomBar"> <property name="sizePolicy"> <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="bottomBarLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Powrót</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Odśwież</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton#refreshButton {
            background-color: #4f545c;
            color: white;
            font-size: 13px;
            font-weight: 600;
            padding: 6px 20px;
            border-radius: 5px;
            border: none;
            min-height: 28px;
            outline: none;
          }
          QPushButton#refreshButton:hover { background-color: #5d6269; }
          QPushButton#refreshButton:pressed { background-color: #6b6f79; }
          QPushButton#refreshButton:disabled { background-color: #40444b; color: #72767d; opacity: 0.7; }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="bottomSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deletePcbButton">
        <property name="text">
         <string>Usuń płytkę PCB</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton#deletePcbButton {
            background-color: #ed4245;
            color: white;
            font-size: 13px;
            font-weight: 600;
            padding: 6px 20px;
            border-radius: 5px;
            border: none;
            min-height: 28px;
            outline: none;
          }
          QPushButton#deletePcbButton:hover { background-color: #c03537; }
          QPushButton#deletePcbButton:pressed { background-color: #a12d2f; }
          QPushButton#deletePcbButton:disabled { background-color: #40444b; color: #72767d; opacity: 0.7; }
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>